src/main/resources/plugin-descriptor.properties

```diff
@@ -27,7 +27,7 @@
 description=Performance Analyzer Plugin
```

Comment:
Why is this checked into version control? Isn't this auto-generated when a plugin is assembled?